src/cbl-mariner/2.0/crossdeps/amd64/Dockerfile

```diff
@@ -2,6 +2,9 @@ FROM mcr.microsoft.com/cbl-mariner/base/core:2.0
 
 RUN tdnf update -y && \
     tdnf install -y \
+        # Needed to install MicroBuild in Azure DevOps pipelines
+        sudo \
+        apt \
```

Comment:
There is no `apt` package in Mariner. It uses `tdnf` as its package manager. The MicroBuild script should support that if it doesn't already.